types/knockout.mapping/knockout.mapping-tests.ts

```diff
@@ -1,105 +1,160 @@
-//variable tests and definitions
-
-let userInputJS: User = { firstName: 'foo', lastName: 'bar', age: 12 };
+////////////////////////////////
+// variable tests and definitions
 interface User {
-    firstName: string;
-    lastName: string;
-    age: number;
+    firstName: string
+    age: number
+    address: Address
+}
+
+interface Address {
+    street: string
 }
 
 interface MappedUser {
-    firstName: KnockoutObservable<string>;
-    lastName: KnockoutObservable<string>;
-    age: KnockoutObservable<number>;
+    firstName: KnockoutObservable<string>
+    age: KnockoutObservable<number>
+    address: MappedAddress
+}
+
+interface MappedAddress {
+    street: KnockoutObservable<string>
+}
+
+interface Car {
+    name: string
+    maintenance: number[]
+    drivers: User[]
+}
+
+interface MappedCar {
+    name: KnockoutObservable<string>
+    maintenance: KnockoutObservableArray<number>
+    drivers: KnockoutObservableArray<MappedUser>
 }
 
 let badUserMappingOptions: KnockoutMappingOptions<User> = {
-    ignore: ['age'],
-    include: ['name'],
-    copy: ['height'],
-    mappedProperties: ['age', 'name'],
+    ignore: ["age"],
+    include: ["name"],
+    copy: ["height"],
+    mappedProperties: ["age", "name"],
     deferEvaluation: false,
-    create: function (options: KnockoutMappingCreateOptions) { }, // $ExpectError
-};
+    create: (options: KnockoutMappingCreateOptions) => { }, // $ExpectError
+}
 
 let userMappingOptions: KnockoutMappingOptions<User> = {
-    ignore: ['age'],
-    include: ['name'],
-    copy: ['height'],
-    mappedProperties: ['age', 'name'],
+    ignore: ["age"],
```

Comment:
Is there a reason to change the quotemark? The Prettier config for DT uses single quotes, so you should stay consistent with that:

https://github.com/DefinitelyTyped/DefinitelyTyped/blob/3ad3eefe64b17cbb77f2a1a3932f72f1be73c028/.prettierrc.json#L7

Reply:
Repo didn't use prettier back when I did the changes. I don't use it my self. Maybe someday. 
I saw that each project used it's own formatting guidelines, so used my opinionated in this one. I prefer double quotes because it is consistent with other common languages like C# and Java. 

In the interest of consistency with this repo and saving time, against my preference, I reverted this changes.